sefaria/model/text.py

```diff
@@ -6058,6 +6092,19 @@ def get_multi_title_regex_string(self, titles, lang, for_js=False, anchored=Fals
 
     # do we want to move this to the schema node? We'd still have to pass the title...
     def get_regex_string(self, title, lang, for_js=False, anchored=False, capture_title=False, parentheses=False):
+        """
+        Given a string with a Ref not in Sefaria format (i.e. "See Genesis 2 3" as opposed to "Genesis 2:3",
```

Comment:
The regex should also work for refs in Sefaria format

Reply:
Also, the search string is not an input to this function. This function only creates the regex which will later be used to search.